cni/pkg/iptables/iptables.go

```diff
@@ -333,6 +333,13 @@ func (cfg *IptablesConfigurator) AppendInpodRules(podOverrides config.PodLevelOv
 		)
 	}
 
+	iptablesBuilder.AppendRule(ChainInpodPrerouting, "nat",
```

Comment:
I don't think you can apply this on the NAT table? Did you mean the mangle table?

Reply:
From dmesg if I try and run this rule I get:

x_tables: ip_tables: DSCP target: only valid in mangle table, not nat